app/src/main/java/com/example/weather/screen/home/WeatherFragment.kt

```diff
@@ -78,29 +108,60 @@ class WeatherFragment private constructor() : BaseFragment<FragmentWeatherBindin
     }
 
     override fun onGetCurrentWeatherSuccess(weather: Weather) {
-        mWeatherTemp = weather
+        if (mWeatherTemp == null) {
+            mWeatherTemp = weather
+        }
         bindDataToView(weather)
     }
 
     private fun bindDataToView(weather: Weather) {
+        if (weather.isFavorite == "false") {
```

Comment:
```suggestion
        if (weather.isFavorite == Constant.FALSE) {
```
nên khai báo "true"/ "false" này thành hằng số để dụng nhiều nơi khác nhau